tensorflow/lite/kernels/internal/BUILD

```diff
@@ -492,7 +492,6 @@ cc_library(
         "reference/integer_ops/fully_connected.h",
         "reference/integer_ops/l2normalization.h",
         "reference/integer_ops/logistic.h",
-        "reference/integer_ops/mean.h",
```

Comment:
it would be easier if we keep this file or please update the tflite-micro usage here: https://github.com/tensorflow/tflite-micro/blob/main/tensorflow/lite/micro/kernels/reduce.cc

thanks!

Reply:
I forgot about the tflite-micro repository. I think it would be best to modify tflite-micro as we need to also adapt the `QuantizedMeanOrSum` calls and it's best to keep both tflite and tflite-micro aligned, we also want to avoid the floating-point operations and divisions there.

What would be the best way, should I create another PR in the tflite-micro repository? Should I include the changes of the common code, like the modification of `QuantizedMeanOrSum`,  in this new PR?

Reply:
tflite-micro is in a different repo right? I guess it's pretty hard to submit two separate PRs at the same time (not sure if it will break anything in the middle)?

I would:
1) keep the `mean.h` file in this PR
2) modify the tflite-micro usage in the next PR
3) delete the `meah.h` file

WDYT?

Reply:
Yes, the problem is that the removal of the `mean.h` is not the only thing that will break tflite-micro when the common files are updated there, the signature changes of `QuantizedMeanOrSum` will also break it.

I think it'd be easier to create one PR here and one PR in tflite-micro that will only cover the change of the non-common code. When the common code is updated in tflite-micro, the code of the second PR should be taken to resolve the issues that arise (we can then close this second PR without merging it).

Otherwise I would need to create three PRs, this one adapted to be backward compatible, a second one in tflite-micro to change the usage and a third one here to remove the dead code (old `QuantizedMeanOrSum` and unused Mean functions).

Reply:
alright then, let me know if the micro changes is submitted